### PR TITLE
add scaffold cart

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: page-cart.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <!-- main posts loop -->
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+    the_content();
+  }
+} else {
+?>
+    <article class="col col12 u-alert"><?php _e('Sorry, no page matched your criteria :{'); ?></article>
```
Comment (mikeyrayvon) [inline page-cart.php]: these classes dont exist in the new grid, ma dude

**Review by mikeyrayvon — COMMENTED** [review]
fix grid classes

**Review by mikeyrayvon — CHANGES_REQUESTED** [review]
fix grid classes

**Comment by mixcas** [comment]
i dont undesrtand this review stuff, but i fixed it

**Comment by varffs** [comment]
Mikey requested changes so he has to mark that they are acceptable before the review turns green (but seems like it can be merged anyhow, that might be a setting we need to update)

**Pull request merged**